amundsen_application/static/css/_popovers.scss

```diff
@@ -30,3 +30,38 @@
 .popover.left .arrow:after {
   border-left-color: $gray-darker;
 }
+
+
+.tooltip-inner {
```

Comment:
I'm concerned about how this will change every other tooltip in the application, and would rather keep this in `EditableText` unless we know this would be acceptable for Design, so that we don't create more work for ourselves having to go back in and fix other tooltips. 

Since we're meeting first thing tomorrow to talk about these exact tickets, can we run this by them?

Reply:
The Tooltip component is only used by the EditableText component and the original styles were already global. I did consider creating a separate `_tooltips.scss` file but given how they are so closely related I figured I could put them in the same file.

I also have the idea of using bootstrap variables to configure tooltips and popovers and deleting most of these rules, but that's out of scope for this task.

Reply:
That's new to me! I had been under the impression `Popover` was an extension of `Tooltip` and therefore shared classes.

Reply:
`Popover` does extend `Tooltip` but the classes are different.